modules/KalturaSupport/components/dualScreen/dualScreen.js

```diff
@@ -308,10 +323,12 @@
 
 					//Attach the primaryPlayerContainer to the primary display
 					var primaryDisplay = this.displays.getPrimary();
+					primaryPlayerContainer.attr('data-display-type','video');
 					primaryDisplay.attachView(primaryPlayerContainer);
 
 					//Attach the secondaryDisplay to the second display
 					var secondaryDisplay = this.displays.getSecondary();
+					secondaryPlayerContainer.attr('data-display-type','presentation');
```

Comment:
vide/presentation display type is not necessarily a fixed state and will change with upcoming dual video feature, and this naming convention will make it quite confusing.
Please consult with @einatr before going along this path to make sure it coincides with her changes.